### PR TITLE
Console output/ printf

### PR DESCRIPTION
**Pull request by ZopharShinta**
## Summary
Added packets for Console Output and Console PrintF

### Issues closed
Client Console Output (packet 17)
Client Console PrintF (packet 19)

Slashcommands added
/consoleOutput messageText
/consolePrint messageText

Added functions to Lua and pages to LuaBot
MapClientSession.ConsoleOutput("Message Text");
MapClientSession.ConsolePrint("Message Text");

### CODE REVIEW

**Comment by broxen** [comment]
This is failing Travis on Linux due to some sort of regression introduced in #719. Further research is needed.

Will review this PR shortly

File: Projects/CoX/Servers/MapServer/DataHelpers.h [hunk 1]
```diff
@@ -98,6 +98,8 @@ void openStore(MapClientSession &sess, int entity_idx);
 void modifyInf(MapClientSession &sess, int amount);
 void sendForceLogout(MapClientSession &cl, QString &player_name, QString &logout_message);
 void sendLocation(MapClientSession &cl, VisitLocation location);
+void sendConsoleOutput(MapClientSession &cl, QString &message);
```
Comment (broxen) [inline Projects/CoX/Servers/MapServer/DataHelpers.h]: So these are the names we've gathered from the client-side:
```
        aCommandNames[17] = "SERVER_SEND_CONSOLE_OUTPUT";
        aCommandNames[19] = "SERVER_CON_PRINTF";
```
But we can probably give them more descriptive/meaningful names.

My understanding is that ConsolePrintF goes to the client chat window, or the in-client console `~` (if open), whereas ConsoleOutput goes to the `-console` window. I'm not sure what nomenclature is best, but let's shoot for something that makes that distinction clear.

Comment (ZopharShinta) [reply Projects/CoX/Servers/MapServer/DataHelpers.h]: Does sendExternalConsoleMessage and sendInternalConsoleMessage make more sense?

Comment (broxen) [reply Projects/CoX/Servers/MapServer/DataHelpers.h]: Maybe `sendClientConsoleOutput()` and `sendDeveloperConsoleOutput()`?

My aim being to describe their intended use, vs characterizing how they're displayed?

**Pull request merged**